### ISSUE TITLE
MOVEXY should respect onLeave

### ISSUE THREAD

**Issue by Ingegneus** [ArnoldSmith86/virtualtabletop#1371]
at the moment MOVEXY doesn't respect the onLeave property of a holder. I can set the face, but I think the default behaviour should be to behave according to onLeave

**Comment by 96LawDawg** [comment]
I disagree.  I think `MOVEXY` should operate like `MOVE`.  Both are methods of automation whereas `onLeave` (as I see it) is used for manual dragging.  You can also easily accomplish the same thing by just adding the `face` property that matches your `onLeave` to the `MOVEXY`.

If your idea is implemented, it would be a breaking change so a file updater would be needed.  (Would probably be easy).  But I think a new property would also be needed.  Something like `ignoreOnLeave` (true or false with a default of false).  Otherwise, the function would always follow the `onLeave` command and `face` would be ignored.  We would need a way to override that.

So for those reasons, I don't think we should do this.

**Comment by robartsd** [comment]
> If your idea is implemented, it would be a breaking change so a file updater would be needed. (Would probably be easy). But I think a new property would also be needed. Something like `ignoreOnLeave` (true or false with a default of false).

I think this would be a new function parameter of `MOVE` and `MOVEXY` not a widget property.

> Otherwise, the function would always follow the `onLeave` command and `face` would be ignored. We would need a way to override that.

Both would be applied, though we'd need to determine which is applied first. We probably could write the code such that any `onLeave` properties are applied before the `face` function parameter is applied.
